### ISSUE TITLE
createFolder(): Return value must be of type bool

### ISSUE THREAD

**Issue by mihakot** [Webklex/php-imap#261]
**Describe the bug**
Webklex\PHPIMAP\Connection\Protocols\ImapProtocol::createFolder(): Return value must be of type bool, null returned

**Used config**
default

**Code to Reproduce**
The troubling code section which produces the reported bug.
```php
        $client = Client::account($account);

        //Connect to the IMAP Server
        $client->connect();

        return $client->createFolder($folder);
```

**Screenshots**
![image](https://user-images.githubusercontent.com/1734182/185783651-d13ffd7c-29c0-4eb4-97f5-6d858e20de8c.png)

**Desktop / Server (please complete the following information):**
- OS: Windows
- PHP: 8.1
- Version 4.0
- Provider YandexMail

**Comment by Webklex** [comment]
Hi @mihakot ,
many thanks for your report. 
I just pushed a patch to cast the return value of several `ImapProtocol::class` methods into a boolean value.

Best regards,

**Comment by mihakot** [comment]
Thanks)
When will published in packagist?

**Comment by Webklex** [comment]
Hi @mihakot ,
this might take a little bit. If you don't mid, I would like to hear your thought regarding the release frequency: #265

Best regards,

**Comment by gruberroland** [comment]
Can this be closed? It is mentioned in [4.0.1](https://github.com/Webklex/php-imap/releases/tag/4.0.1) as fixed.
